app/services/hyrax/collections/permissions_service.rb

```diff
@@ -81,7 +81,7 @@ def self.can_deposit_in_collection(user:, collection:)
         return false unless user && collection
         template = Hyrax::PermissionTemplate.find_by!(source_id: collection.id)
         return true if access_as_user?(user: user, template: template)
-        return true if access_through_group?(groups: user.user_groups, template: template)
+        return true if access_through_group?(groups: user.ability.user_groups, template: template)
```

Comment:
I suspect you have outdated code.  There was an abilities refactor that significantly changed this service.  The method `access_through_group?` doesn't exist anymore.  Let me know if you can't do what you need with the refactored service.